lara-typescript/src/page-item-authoring/common/components/interactive-iframe.tsx

```diff
@@ -25,13 +21,38 @@ interface IFramePhoneParentEndpoint {
 
 export const InteractiveIframe: React.FC<Props> = (props) => {
   const {
-    src, width, height, initMsg, onAuthoredStateChange, resetCount,
-    onSupportedFeaturesUpdate, onHeightChange, onSetIFrameRef,
-    aspectRatio, aspectRatioMethod
+    src, width, initMsg, onAuthoredStateChange, resetCount,
+    authoredAspectRatio, authoredAspectRatioMethod
   } = props;
 
   const iframe = useRef<HTMLIFrameElement|null>(null);
-  onSetIFrameRef(iframe.current);
+
+  // FIXME: The interactive sizing computation at runtime is currently handled
+  // by the setSize method in interactives-sizing.js
+  // That code can't be used here, because the jQuery changes to the iframe
+  // conflict with the React management of the iframe element.
+  // We could:
+  // - duplicate the sizing code here
+  // - abstract it so it can be shared by both the runtime and authoring.
+  // - move the runtime iframe rendering into React, so both authoring and
+  //   runtime use the interactiveIframe component
+  // The last option is the best for maintainability, but it will slow down the
+  // page load since the iframes won't start loading until the javascript is loaded
+  // So probably the best option is to abstract that sizing code so it can be
+  // used by both jQuery and React
+  const [height, setHeight] = useState<number|string|null>(null);
+  const [aspectRatio, setAspectRatio] = useState<number>(authoredAspectRatio);
+
+  const handleHeightChange = (newHeight: number | string) => {
+    setHeight(newHeight);
+  };
+
+  const handleSupportedFeatures = (info: any) => {
```

Comment:
This doesn't handle the call to `setAuthoringSupported(!!info.features.authoredState)` that was deleted in this PR in this file:

lara-typescript/src/page-item-authoring/common/components/interactive-authoring.tsx

Reply:
Good catch. next commit takes care of this.  However it looks like it was not being used, but we might use it in the future.